src/sas/qtgui/MainWindow/MainWindow.py

```diff
@@ -117,7 +117,7 @@ def run_sasview():
     from twisted.internet import reactor
 
     # Show the main SV window
-    mainwindow = MainSasViewWindow()
+    mainwindow = MainSasViewWindow(filepath=filepath)
```

Comment:
Rather than adding a `filepath` argument to `MainSasViewWindow` you may want to trigger a QDropEvent after the GUI startup is complete.